Linux_kernel_diagram.dot

```diff
@@ -385,7 +385,7 @@ digraph "Linux_kernel_diagram" {
 		functions [
 			color = gray,
 			tooltip = "Columns represent main functionalities of the kernel",
-			URL = "http://www.makelinux.net/ldd3/chp-1-sect-2.shtml",
+			URL = "https://makelinux.github.io/ldd3/chp-1-sect-2.shtml",
```

Comment:
This URL from the [link in the "functions" header](https://makelinux.github.io/kernel/diagram/) (diagram) and the second link [in the menu labelled "Splitting the kernel"](https://makelinux.github.io/kernel/map/) (map) points to nowhere. Can you suggest an appropriate replacement? :confused:

Reply:
fixed: https://www.oreilly.com/library/view/linux-device-drivers/0596000081/ch01s02.html